pdal/PointView.hpp

```diff
@@ -301,6 +319,7 @@ class PDAL_DLL PointView : public PointContainer
     std::queue<PointId> m_temps;
     SpatialReference m_spatialReference;
     std::map<std::string, std::unique_ptr<TriangularMesh>> m_meshes;
+    std::map<std::string, std::unique_ptr<Rasterd>> m_rasters;
```

Comment:
Would the existing GDAL raster bands also get created and added in such a manner? Maybe not, as those are bands for a single raster out.

Reply:
I've reworked the GDALWriter to use the Raster class, but right now it's all still self-contained.  That bit is a work in progress. I have mixed feelings about all of this, as I did with the mesh code. I'm just not sure where it ends.